executor.sh

```diff
@@ -25,4 +25,32 @@ echo "--> Copying ${SIDECAR_EXECUTOR_PATH} to sandbox ${MESOS_SANDBOX}"
 executor="${MESOS_SANDBOX}/`basename ${SIDECAR_EXECUTOR_PATH}`"
 cp $SIDECAR_EXECUTOR_PATH $executor
 echo "--> Starting ${executor}"
-exec $executor -logtostderr=true
+
+function cleanup {
```

Comment:
I wonder if we should move this to the custom script, as we do now to load the env.
I'm ok merging this as it's, but we may want a more generic `function onExit`

Reply:
Technically, this project spins up Docker containers, so it should also make sure it stops them. The custom script doesn't need to be aware of Docker.